server/src/main/java/org/apache/iotdb/db/engine/flush/MemTableFlushTask.java

```diff
@@ -237,7 +237,6 @@ public void run() {
                   (Pair<TVList, MeasurementSchema>) task;
               IChunkWriter seriesWriter = new ChunkWriterImpl(encodingMessage.right);
               writeOneSeries(encodingMessage.left, seriesWriter, encodingMessage.right.getType());
-              seriesWriter.sealCurrentPage();
```

Comment:
this code can not delete.  because this  method make encode data...
this will lead to encode problem...